src/data-loader.test.ts

```
@@ -1902,4 +1902,18 @@ describe('calculateCostForEntry', () => {
 			expect(result).toBe(-0.01);
 		});
 	});
+
+	describe('offline mode', () => {
```

ESLint

[error] Unsafe call of a(n) `error` type typed value.

(ts/no-unsafe-call)